File: 1146.py
lista=[]
a = 0
while True:
    n = int(input())
    if n == 0:
        break
    lista+=[n]
for j in range(0, len(lista)):
    for i in range(1, lista[j]+1):
        print(i,end='') if j == len(lista) - 1 and i == lista[j] or i == lista[j] else print(i,end=' ')
    if j != len(lista)-1:
        print('\n',end='')
print()

File: 1146.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        String numeros = "";
        while(n!=0){
                for(int i = 1; i < n+1; i++){
                        if(i == n){
                                numeros+=i;
                        }
                        else{
                                numeros+=i;
                                numeros+=" ";
                        }
                }
                System.out.println(numeros);
                numeros = "";
                n = input.nextInt();
        }
}
}
